weave

```diff
@@ -979,6 +1007,7 @@ case "$COMMAND" in
         shift $CIDR_COUNT
         [ $# -eq 1 ] || usage
         CONTAINER=$(container_id $1)
+        ipam_cidrs $CONTAINER $CIDR_ARGS
```

Comment:
I'm fixing the bug that DELETE was never called, but it still seems easiest to go through the allocation process first, since it will return the previously allocated address when you do things in the right order, and will only temporarily allocate an unnecessary address when you do 'detach' with no preceding 'attach'.

The alternative is to have both POST and GET variants of `ipam_cidrs`, and handle the different error cases.